src/Module/Register.php

```diff
@@ -426,4 +427,19 @@ private function sendNotification(array $user, string $event)
 			]);
 		}
 	}
+	public static function getRegisterPolicy(): int
```

Comment:
Class is already called `Register`.
```suggestion
	public static function getPolicy(): int
```

Reply:
Done